ca_tools/scripts/pos_controller_pid.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+import rospy
+import math
+from nav_msgs.msg import Odometry
+from geometry_msgs.msg import Twist
+from tf.transformations import euler_from_quaternion
+
+
+
+class Execute_Draw_Square:
+
+    # Time variables
+    now=float
```

Comment:
Variables that are used throughout the class, need to have `self.` appended. Check [here](https://medium.com/quick-code/understanding-self-in-python-a3704319e5f0).